src/Config/ProjectsConfiguration.php

```diff
@@ -60,6 +60,7 @@ public function getConfigTreeBuilder()
                                         ->arrayNode('php')->prototype('scalar')->defaultValue([])->end()->end()
                                         ->arrayNode('services')->prototype('scalar')->defaultValue([])->end()->end()
                                         ->scalarNode('target_php')->defaultNull()->end()
+                                        ->scalarNode('phpunit_version')->defaultNull()->end()
```

Comment:
Why not default 8 here? And no default in the twig template?

Reply:
I noticed this file afterwards, am on my phone now so can't change now, might change later if doing so makes more sense (I'm not sure, will think about it)

Reply:
@OskarStark I gave this a second thought, and we might have to reintroduce conditions as above later, ending up with something like this: `{{ phpunit_version|default('7.1' in php ? '7' : '8')}}`. If we put the default in the config, then we would not know from twig if what is specified is the default value or something we explicitely set, meaning we would have no way to override that kind of logic I think we might reintroduce. Can we keep things as is?

Reply:
That makes sense 👍🏻
Sure we keep it like this

@Soullivaneuh please merge